docker-images/Makefile

```diff
@@ -0,0 +1,5 @@
+REPOSITORY_URI ?= awslabs/s2n-dev
+
+build:
+	@cp -R ../codebuild/ ./codebuild/
```

Comment:
😢  but works for now.

Reply:
yeah it's kind of unfortunate but since the Dockerfile needs to be inside the context I'm not sure there's a way around it. We can revisit to clean it up if something comes up.